### ISSUE TITLE
bug parsing mm registers

### ISSUE THREAD

**Issue by bchurchill** [StanfordPL/x64asm#195]
This probably got introduced when I started work on the parser.

```
$ asm
paddq %mm2, %mm2                  

Parse error encountered.
Failed to parse line 1: paddq %mm2, %mm2
Message: Could not parse operand 1: Don't know how to parse this!
```

**Comment by bchurchill** [comment]
Also, all the st registers.  Downgrading to low priority since I'm realizing nobody uses these features at all.

**Comment by stefanheule** [comment]
Fixed in #214
